seacrowd/sea_datasets/ud/ud.py

```diff
@@ -0,0 +1,176 @@
+
+_DATASETNAME = "ud-v2.13"
+
+_SUBSETS = {"id_gsd" : "UD_Indonesian-GSD", 
+            "id_csui": "UD_Indonesian-CSUI", 
+            "id_pud" : "UD_Indonesian-PUD", 
+            "vi_vtb": "UD_Vietnamese-VTB"}
+
+_DESCRIPTION = """\
+Universal Dependencies (UD) is a project that is developing cross-linguistically consistent treebank annotation
+ for many languages, with the goal of facilitating multilingual parser development, cross-lingual learning, and 
+ parsing research from a language typology perspective. The annotation scheme is based on an evolution of (universal)
+   Stanford dependencies (de Marneffe et al., 2006, 2008, 2014), Google universal part-of-speech tags 
+   (Petrov et al., 2012), and the Interset interlingua for morphosyntactic tagsets (Zeman, 2008). 
+   The general philosophy is to provide a universal inventory of categories and guidelines to facilitate consistent
+     annotation of similar constructions across languages, while allowing language-specific extensions when necessary.
+"""
+
+_HOMEPAGE = "https://lindat.mff.cuni.cz/repository/xmlui/handle/11234/1-5287"
+
+_LICENSE = "Apache license 2.0 (apache-2.0)"
+
+_URLS = {
+    "ud-v2.12": "https://lindat.mff.cuni.cz/repository/xmlui/bitstream/handle/11234/1-5150/ud-treebanks-v2.12.tgz?sequence=1&isAllowed=y",
+    "ud-v2.13": "https://lindat.mff.cuni.cz/repository/xmlui/bitstream/handle/11234/1-5287/ud-treebanks-v2.13.tgz?sequence=1&isAllowed=y"
+}
+
+_SUPPORTED_TASKS = [Tasks.POS_TAGGING]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+
+
+
+class UDDataset(datasets.GeneratorBasedBuilder):
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=f"{_DATASETNAME}",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_seq_label",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd Seq Label schema",
+            schema="seacrowd_seq_label",
+            subset_id=f"{_DATASETNAME}",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+
+    UPOS_TAGS = ["ADJ", "ADP", "ADV", "AUX", "CCONJ", "DET", "INTJ", "NOUN", "NUM", "PART", "PRON", "PROPN", "PUNCT", "SCONJ", "SYM", "VERB", "X"]
+
+    def _info(self) -> datasets.DatasetInfo:
+        self.config.schema = "seacrowd_seq_label"
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    # metadata
+                    "sent_id": datasets.Value("string"),
+                    "text": datasets.Value("string"),
+                    "text_en": datasets.Value("string"),
+                    # tokens
+                    "id": [datasets.Value("string")],
```

Comment:
should use datasets.Sequence(datasets.Value("string")) instead.

Reply:
also for other features.

Reply:
Fixed